### PR TITLE
Update TinyIDTest.php

### PR DESCRIPTION
**Pull request by ternson**

### CODE REVIEW

File: tests/Unit/TinyIDTest.php [hunk 1]
```diff
@@ -117,4 +117,11 @@ public function failuresOnDecodingWithCharacter()
             self::assertEquals('cannot decode string with characters not in dictionary', $e->getMessage());
         }
     }
+    
+    public function dictionaryWithRepeatedSymbols()
```
Comment (krowinski) [inline tests/Unit/TinyIDTest.php:121]: missing test annotate in comment - btw this will throw exception  as I don't allow duplicate keys
look in failuresProvider ['aa', 'dictionary contains duplicated characters'],
